XamlControlsGallery/ControlPages/ListViewPage.xaml

```diff
@@ -109,99 +171,324 @@
+                    SelectionMode="Single" 
+                    BorderThickness="1"
+                    BorderBrush="DimGray"
+                    MinWidth="350" 
+                    Height="400"
+                    DragItemsStarting="Source_DragItemsStarting"
+                    DragOver="Source_DragOver"
+                    Drop="Source_Drop"
+                    Margin="15"
+                    />
+                <ListView
+                    x:Name="DragDropListView2"
+                    ItemTemplate="{StaticResource ContactListViewTemplate}"
+                    CanDragItems="True"
+                    CanReorderItems="True"
+                    AllowDrop="True"
+                    DragOver="Target_DragOver"
+                    Drop="Target_Drop"
+                    DragItemsStarting="Target_DragItemsStarting"
+                    DragEnter="Target_DragEnter"
+                    SelectionMode="Single" 
+                    BorderThickness="1"
+                    BorderBrush="DimGray"
+                    MinWidth="350" 
+                    Height="400"/>
+                </StackPanel>
+            </Grid>
+            
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;!-- Full code to implement dragging and dropping between lists can be found in the 
+source code for the XAML Controls Gallery. -->
+
+&lt;!-- The following properties should be added to the ListView definitions to build this sample. --&gt;
 &lt;ListView CanDragItems="True" CanReorderItems="True" AllowDrop="True"
-    SelectionMode="$(SelectionMode)" /&gt;
+    SelectionMode="Single" /&gt;
+    
+&lt;!-- DataTemplate used: --&gt;
+
+&lt;DataTemplate x:Key="ContactListViewTemplate" x:DataType="local1:Contact">
+    &lt;Grid&gt;
+        &lt;Grid.RowDefinitions&gt;
+            &lt;RowDefinition Height="*"/&gt;
+            &lt;RowDefinition Height="*"/&gt;
+        &lt;/Grid.RowDefinitions&gt;
+        &lt;Grid.ColumnDefinitions&gt;
+            &lt;ColumnDefinition Width="Auto"/&gt;
+            &lt;ColumnDefinition Width="*"/&gt;
+        &lt;/Grid.ColumnDefinitions&gt;
+        &lt;Ellipse x:Name="Ellipse"
+                    Grid.RowSpan="2"
+                    Width ="32"
+                    Height="32"
+                    Margin="6"
+                    VerticalAlignment="Center"
+                    HorizontalAlignment="Center"
+                    Fill="Gray"/&gt;
+        &lt;TextBlock Grid.Column="1"
+                    Text="{x:Bind Name}"
+                    x:Phase="1"
+                    Style="{ThemeResource BaseTextBlockStyle}"
+                    Margin="12,6,0,0"/&gt;
+        &lt;TextBlock  Grid.Column="1"
+                    Grid.Row="1"
+                    Text="{x:Bind Company}"
+                    x:Phase="2"
+                    Style="{ThemeResource BodyTextBlockStyle}"
+                    Margin="12,0,0,6"/&gt;
+    &lt;/Grid>
+&lt;/DataTemplate&gt;
```

Comment:
I feel like you don't need this because this data template is already shown above.